### PR TITLE
[L1T] switch to GeomeryDB

### PR DESCRIPTION
**Pull request by cecilecaillol**
#### PR description:

Addresses issue https://github.com/cms-sw/cmssw/issues/31113 ("Configuration.StandardSequences.Geometry_cff was removed")

### CODE REVIEW

**Comment by cecilecaillol** [comment]
please test

**Comment by cecilecaillol** [comment]
+l1

**Comment by perrotta** [comment]
+1
- Test configs updated

File: EventFilter/L1GlobalTriggerRawToDigi/test/L1GtPacker_cfg.py [hunk 1]
```diff
@@ -276,7 +276,7 @@
 # load and configure modules via Global Tag
 # https://twiki.cern.ch/twiki/bin/view/CMS/SWGuideFrontierConditions
 
-process.load('Configuration.StandardSequences.Geometry_cff')
+process.load("Configuration.StandardSequences.GeometryDB_cff")
 process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
 
 process.GlobalTag.globaltag = useGlobalTag+'::All'
```
Comment (mmusich) [inline EventFilter/L1GlobalTriggerRawToDigi/test/L1GtPacker_cfg.py:282]: just wanted to mention that all the `::All` here are relics from Cond DB version 1 and they certainly cannot work with the current DB infrastructure.

File: EventFilter/L1GlobalTriggerRawToDigi/test/L1GtRecord_cfg.py [hunk 2]
```diff
@@ -114,7 +114,7 @@
 # load and configure modules via Global Tag
 # https://twiki.cern.ch/twiki/bin/view/CMS/SWGuideFrontierConditions
 
-process.load('Configuration.StandardSequences.Geometry_cff')
+process.load("Configuration.StandardSequences.GeometryDB_cff")
 process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
 
 process.GlobalTag.globaltag = useGlobalTag+'::All'
```
Comment (mmusich) [inline EventFilter/L1GlobalTriggerRawToDigi/test/L1GtRecord_cfg.py:120]: also here

File: EventFilter/L1GlobalTriggerRawToDigi/test/L1GtTriggerMenuLite_cfg.py [hunk 3]
```diff
@@ -33,7 +33,7 @@
 # load and configure modules via Global Tag
 # https://twiki.cern.ch/twiki/bin/view/CMS/SWGuideFrontierConditions
 
-process.load('Configuration.StandardSequences.Geometry_cff')
+process.load("Configuration.StandardSequences.GeometryDB_cff")
 process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
 
 process.GlobalTag.globaltag = useGlobalTag+'::All'
```
Comment (mmusich) [inline EventFilter/L1GlobalTriggerRawToDigi/test/L1GtTriggerMenuLite_cfg.py:39]: here.

File: L1Trigger/CSCTrackFinder/test/csctfEfficiencySim_cfg.py [hunk 4]
```diff
@@ -35,7 +35,7 @@
 ##############
 process.load("Configuration.StandardSequences.Services_cff")
 process.load("Configuration.StandardSequences.MagneticField_cff")
-process.load("Configuration.StandardSequences.Geometry_cff")
+process.load("Configuration.StandardSequences.GeometryDB_cff")
 process.load("Configuration.StandardSequences.Simulation_cff")
 process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
 process.GlobalTag.globaltag ='START52_V1::All'
```
Comment (mmusich) [inline L1Trigger/CSCTrackFinder/test/csctfEfficiencySim_cfg.py:41]: here.

File: L1Trigger/GlobalTriggerAnalyzer/test/L1GtPackUnpackAnalyzer_cfg.py [hunk 5]
```diff
@@ -109,7 +109,7 @@
 # load and configure modules via Global Tag
 # https://twiki.cern.ch/twiki/bin/view/CMS/SWGuideFrontierConditions
 
-process.load('Configuration.StandardSequences.Geometry_cff')
+process.load("Configuration.StandardSequences.GeometryDB_cff")
 process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
 
 process.GlobalTag.globaltag = useGlobalTag+'::All'
```
Comment (mmusich) [inline L1Trigger/GlobalTriggerAnalyzer/test/L1GtPackUnpackAnalyzer_cfg.py:115]: here.

File: L1Trigger/GlobalTriggerAnalyzer/test/L1GtTechTrigL1GtEmulTrigReport_cfg.py [hunk 6]
```diff
@@ -210,7 +210,7 @@
 # load and configure modules via Global Tag
 # https://twiki.cern.ch/twiki/bin/view/CMS/SWGuideFrontierConditions
 
-process.load('Configuration.StandardSequences.Geometry_cff')
+process.load("Configuration.StandardSequences.GeometryDB_cff")
 process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
 
 process.GlobalTag.globaltag = useGlobalTag + '::All'
```
Comment (mmusich) [inline L1Trigger/GlobalTriggerAnalyzer/test/L1GtTechTrigL1GtEmulTrigReport_cfg.py:216]: here.

File: L1Trigger/RegionalCaloTrigger/test/testRCTEmulator.py [hunk 7]
```diff
@@ -16,7 +16,7 @@
 )
 
 # standard includes
-process.load("Configuration.StandardSequences.Geometry_cff")
+process.load("Configuration.StandardSequences.GeometryDB_cff")
 process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
 process.GlobalTag.globaltag = "IDEAL_30X::All"
```
Comment (mmusich) [inline L1Trigger/RegionalCaloTrigger/test/testRCTEmulator.py:21]: here.

File: L1Trigger/RegionalCaloTrigger/test/testTPGManipulation_cfg.py [hunk 8]
```diff
@@ -16,7 +16,7 @@
 )
 
 # standard includes
-process.load("Configuration.StandardSequences.Geometry_cff")
+process.load("Configuration.StandardSequences.GeometryDB_cff")
 process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
 process.GlobalTag.globaltag = "IDEAL_30X::All"
```
Comment (mmusich) [inline L1Trigger/RegionalCaloTrigger/test/testTPGManipulation_cfg.py:21]: and here.

**Pull request merged**